src/biocommons/seqrepo/fastadir/fastadir.py

```diff
@@ -123,11 +120,12 @@ def fetch(self, seq_id, start=None, end=None):
         fabgz = self._open_for_reading(path)
         return fabgz.fetch(seq_id, start, end)
 
+    @functools.lru_cache(maxsize=os.environ.get("SEQREPO_LRU_CACHE_MAXSIZE"))
```

Comment:
Suggestion: 1) be explicit about a default cache size, 2) fetch the env var once, and include the default there. Specifically:

Near the top (below imports), add `cache_maxsize = os.environ.get("SEQREPO_LRU_CACHE_MAXSIZE", 1000)`.

Then, replace both decorator lines with `@functools.lru_cache(maxsize=cache_maxsize)`